docs/try-viam/rover-resources/rover-tutorial-fragments.md

```diff
@@ -33,18 +33,18 @@ Click **Save Config** to save the new configuration.
 
 {{<imgproc src="try-viam/rover-resources/fragments/fragment_configuration.png" resize="1200x" alt="Fragment configuration">}}
 
-The fragment adds the following components to your robot's JSON configuration:
+The fragment adds the following components to your robot's JSON configuration.
+For information about how you would configure a component yourself if you weren't using the fragment, click the links on each component below.
+To see the pin numbers and other values specific to this fragment, [see the components on the configuration page](#see-the-components-on-the-configuration-page).
 
-* [Board component](/components/board/), which is the Raspberry Pi.
-  * Within the board component attributes, digital interrupts: "re" to pin "37" and "le" to pin "35" and I2Cs: name "default_i2c_bus" and bus "1".
-* Right gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Renc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "16 GPIO 23", B/In2: "18 GPIO 24", PWM: "22 GPIO 25".
-  * Depends on local and Renc.
-* Left gpio [motor](/components/motor/).
-  * Within the motor attributes, board: "local", encoder: "Lenc", ticks per rotation: "1992".
-  * Within the component pin assignment, type: In1/In2, A/In1: "11 GPIO 17", B/In2: "13 GPIO 27", PWM: "15 GPIO 22".
-  * Depends on local and Lenc.
+- A [board component](/components/board/pi/) named `local` representing the Raspberry Pi
```

Comment:
This no longer has any relation to the first sentence in the paragraph. Move the two sentences you've added to below the list.

Reply:
Also this list is not fully accurate. It also adds a base_remote_control service, and an input controller

Reply:
done